src/main/java/racing/car/Position.java

```diff
@@ -0,0 +1,39 @@
+package racing.car;
+
+public class Position {
+
+    private int value;
+
+    public Position(int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public Position increase() {
+        this.value++;
+        return this;
+    }
```

Comment:
이런경우 참조객체로써 값이 변화되면 같이 변형되는 잠재적 문제가 있습니다.

```java
        Car car1 = new Car("ㅁㅁ", 0);
        car1.isGo(7);
        Car car2 = car1;
        car2.isGo(7);

        assertThat(car1.getPosition()).isEqualTo(2);
        assertThat(car2.getPosition()).isEqualTo(2);
```

위 테스트는 통과할까요?

문맥적 흐름에 따라 car1이 한번전진하고 car2에 복사한다음 car2를 전진시켰으니, car2는 포지션이 2이고 car1은 1이어야할것 같지만, 둘다 포지션은 2일거에요!

그렇기 때문에 값객체로 활용하는 것들은 final 불변으로 관리한 뒤, 값이 바뀔때마다 새로운 값을 가진 값객체를 할당해주는 편입니다.

```suggestion
    public Position increase() {
        return new Position(this.value+1);
    }
```